api/v1alpha1/storageconsumer_types.go

```diff
@@ -42,7 +42,18 @@ type StorageConsumerSpec struct {
 	Enable bool `json:"enable,omitempty"`
 	// StorageQuotaInGiB describes quota for the consumer
 	// +optional
-	StorageQuotaInGiB int `json:"storageQuotaInGiB,omitempty"`
+	StorageQuotaInGiB          int         `json:"storageQuotaInGiB,omitempty"`
+	RadosNamespace             string      `json:"radosNamespace,omitempty"`
+	SubVolumeGroup             string      `json:"subVolumeGroup,omitempty"`
+	StorageClasses             []ClassSpec `json:"storageClasses,omitempty"`
+	VolumeSnapshotClasses      []ClassSpec `json:"volumeSnapshotClasses,omitempty"`
+	VolumeGroupSnapshotClasses []ClassSpec `json:"volumeGroupSnapshotClasses,omitempty"`
+	ClientProfiles             []string    `json:"clientProfiles,omitempty"`
+}
+
+// Represents the struct to represent a class
+type ClassSpec struct {
+	Name string `json:"name,omitempty"`
```

Comment:
Please have different types for the different types of classes

Reply:
can be done.